yb-voyager/src/srcdb/data/yb-voyager-gather-metadata-and-stats.psql

```diff
@@ -0,0 +1,117 @@
+/*
```

Comment:
As suggested in one of the other comments:
- provide each query in a separate file;
- provide a bash script to run `psql SCRIPT_FILE` for each query;
- the bash script also calls `pg_dump` to dump schema in a single file. The command arguments should be same as used by the `export schema` command.

Reply:
change done